tools/topology/topology2/include/pipelines/cavs/passthrough-copier-be.conf

```diff
@@ -0,0 +1,104 @@
+#
+# BE DAI passthrough pipeline
+#
+# All attributes defined herein are namespaced by alsatplg to
+# "Object.Pipeline.passthrough-copier-be.N.attribute_name"
+#
+# Usage: passthrough-copier-be pipeline object can be instantiated as:
+#
+# Object.Pipeline.passthrough-copier-be."N" {
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."passthrough-copier-be" {
```

Comment:
can we call it dai-copier-buffer.conf? all copiers are capable of format conversion. So its not just passthrough anymore

Reply:
Ack,